projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.ts

```diff
@@ -10,33 +18,70 @@ import { ReadResource, SystemPropertyDefinition } from '@dasch-swiss/dsp-js';
 })
 export class PropertyViewComponent implements OnInit {
 
-  @ViewChild('displayEdit') displayEditComponent: DisplayEditComponent;
+    @ViewChild('displayEdit', { static: false}) displayEditComponent: DisplayEditComponent;
+    @ViewChild('addValue', { static: false}) addValueComponent: AddValueComponent;
```

Comment:
the parameter { static: false} is not required anymore in Angular 9 (https://angular.io/guide/static-query-migration)

Reply:
thanks :) removed in c212919